src/model/add-dots.ts

```diff
@@ -0,0 +1,53 @@
+// Current fix for incorrect use of Middot
+
+const transformDot = (html: string): string => {
```

Comment:
The regex is using the `g` flag so it should really be `transformDots` as it will handle _muiltiple_ dots